src/app/lokalisaatio/route.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  configuration,
+  isDev,
+  isLocalhost,
+  isProd,
+} from '../lib/configuration';
+import finnishTranslations from './fi.json';
+import swedishTranslations from './sv.json';
+import englishTranslations from './en.json';
+
+const REVALIDATE_TIME_SECONDS = 60 * 60 * 2;
+
+async function getTranslations(lng: string) {
+  const res = await fetch(`${configuration.lokalisaatioUrl}${lng}`, {
+    next: { revalidate: REVALIDATE_TIME_SECONDS },
+  });
+  const data = await res.json();
+  const translations: Record<string, string> = {};
+  for (const translation of data) {
+    translations[translation.key] = translation.value;
+  }
+  return translations;
+}
+
+function getTranslationsFromFile(lng: string) {
+  console.log('getTranslationsFromFile ' + lng);
+  switch (lng) {
+    case 'sv':
+      return swedishTranslations;
+    case 'en':
+      return englishTranslations;
+    default:
+      return finnishTranslations;
+  }
+}
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const lng = searchParams.get('lng') || 'fi';
+  if (!isProd && isDev && isLocalhost) {
```

Comment:
Jos `isDev`, niin silloin myös aina `!isProd`.
```suggestion
  if (isDev && isLocalhost) {
```